src/devices/Ft4222/Ft4222Gpio.cs

```diff
@@ -0,0 +1,184 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Device.Ft4222;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Threading;
+
+namespace System.Device.Gpio
+{
+    /// <summary>
+    /// GPIO driver for the FT4222
+    /// </summary>
+    public class Ft4222Gpio : GpioDriver
+    {
+        private const int PinCountConst = 4;
+
+        private SafeFtHandle _ftHandle = new SafeFtHandle();
```

Comment:
I don't think the initializer should be required since you initialize it in the constructor

Reply:
Absolutely, correcting SPI and I2C as well.